examples/nasjob-example-envelopenet.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: nasenvelopenet
+spec:
+  studyName: nasenvelopenet
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: MeanSSS
+  optimizationgoal: 0.99
+  requestcount: 6
+  metricsnames:
+    - accuracy
+  nasConfig:
+    graphConfig:
+      numLayers: 3
+      inputSize:
+        - 32
+        - 32
+        - 3
+      outputSize:
+        - 10
+    operations:
+      - operationType: envolpenet
+        parameterconfigs:
+          - name: envelopecell
+            parametertype: categorical
+            feasible:
+              list:
+              - "1"
+              - "3"
+              - "5"
+              - "3sep"
+              - "5sep"
+              - "7sep"
+          - name: layers_per_stage
+            parametertype: categorical
+            feasible: 
+              list:
+              - "3"
+              - "3"
+              - "3"
+          - name: max_layers_per_stage
```

Comment:
`max_layers_per_stage` is the hyper-parameter of the NAS algorithm itself. Not sure if it should be here, perhaps, you are handling it in your script to treat it as something static.

Reply:
NAS and hyperparameter tuning should not coexist. Please make it a parameter in Suggestion parameters or graph config